docs/registration_part_2/data-models.md

```diff
@@ -0,0 +1,29 @@
+# Data Model Guidelines
+
+## Overview
+
+Ideally, our Django models will support both v1 and v2. We'll achieve this mainly with use of `null=True`, but in some cases more may be required.
+
+## Adding New Tables and Columns
+
+If you need to add something new, you can directly edit the model files (in `bc_obps/registration/models`):
+
+- If you're creating a new table, create a new file for it
+- If you're adding a column to an existing table, it needs to be nullable so that v1 can continue to use the existing table. Set `(null=True)` and `(blank=True)` (null is for the db and blank is for django forms)
+- Create and run migrations
+- Add tests to the v2 test folder. The testing commands in [test.md](../backend/test.md) run the tests for both v1 and v2
+
+## Removing Columns
+
+If data is no longer necessary for v2 but existed in v1:
+
+- Set the column to nullable and blank in the model file
+- Run migrations
+- No test updates needed--we don't test if fields are mandatory or not
```

Comment:
It would be good to check if that field is being referenced anywhere in the codebase. If it is, we must remove that reference to avoid breaking the app.